src/cloudai/__main__.py

```diff
@@ -236,7 +238,7 @@ def handle_dry_run_and_run(mode: str, system: System, tests: List[Test], test_sc
     if mode == "run":
         logging.info(
```

Comment:
Do you think it will be better to move this log line into `main()`? Passing this log-only parameter feels weird to me.

If you don't think it is a good idea, let's at least move `log_file` to the end of list: `def handle_dry_run_and_run(..., log_file: str) -> None:`